acquia/acquia_cms
# ACMS-893: Update places address & telephone on search search page

**Motivation**

Fixes #[ACMS-893]

**Proposed changes**

Update place search result view mode to show address and telephone numer

**Alternatives considered**

NA
**Testing steps**


- Install site with acquia_cms profile
- Enable starter module 
- Visit search page
- Look for places, it should show address along with telephone number 

**Merge requirements**
